.github/workflows/gh_pages.yml

```diff
@@ -41,5 +41,4 @@ jobs:
           git config user.name github-actions
           git config user.email github-actions@github.com
           git add .
-          git commit -m "Add documentation"
-          git push
+          git diff --exit-code --cached || git commit -m "Add documentation" && git push
```

Comment:
Is this being committed to a local copy of the code or into the master?

Reply:
Is the push against the ghpages branch?

Reply:
It is. The workflow checks out the gh_pages branch on lines 27-31.

Reply:
Nice. Then I'll accept and merge.
Thanks for fixing this.